### PR TITLE
Web server/artifacts graphql tests

### PR DESCRIPTION
**Pull request by MooseJ**
Github Bug/Issue: Fixes #???? 
Not sure what issue this should reference. This is just adding more graphql tests

### CODE REVIEW

**Comment by djmitche** [comment]
No need to reference anything, then.  You can add a silent changelog with `yarn changelog --silent --no-bug`.  It looks like the Python client error was due to a connection failure to pypi, so unrelated to your PR.

**Comment by MooseJ** [comment]
> So I think we can omit the call to `getArtifact` in the `artifact` loader. And, I think many of the properties of `Artifact` such as `storageType` are not actually set on the response to that query.

Is this something that you want me to do as a part of this pr?

> Basically, I'm looking at this code now and wondering how it ever worked at all! It might be good to have a deeper look and maybe add some more tests here to uncover any other issues.

Same question here, did you want me to add tests for the artifact loaders in this pr?

File: services/web-server/src/loaders/artifacts.js [hunk 1]
```diff
@@ -62,7 +62,12 @@ module.exports = ({ queue }, isAuthed, rootUrl) => {
 
           return new Artifact(
             taskId,
-            withUrl(queue.getArtifact, taskId, artifact, runId),
+            withUrl({
+              method: queue.getArtifact,
+              taskId,
+              artifact,
+              runId,
+            }),
```
[the remark this answers is not in the document]
Comment (MooseJ) [reply services/web-server/src/loaders/artifacts.js]: I can test this bug locally to see if its fixed with this

**Comment by MooseJ** [comment]
https://github.com/taskcluster/taskcluster/blob/0c0a3cee94c33ceb0830b65ef59bb41c8dff288f/services/web-server/src/graphql/Artifacts.graphql#L85
One other thing I noticed while writing tests is that this latestArtifact query may not be used, but I'm also just basing that on this not being handled in the resolvers and loaders. Is this something that can be removed, should this be kept for future functionality, or does a loader and resolver needed to be added for this

**Comment by djmitche** [comment]
I'll defer to @helfi92 as to whether this should merge as-is, and how the other questions should be handled.

File: services/web-server/src/loaders/artifacts.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (helfi92) [reply services/web-server/src/loaders/artifacts.js]: amazing!

**Pull request merged**